ArcGISRuntimeSDKQt_CppSamples/Maps/GenerateOfflineMapLocalBasemap/GenerateOfflineMapLocalBasemap.cpp

```diff
@@ -118,8 +118,8 @@ void GenerateOfflineMapLocalBasemap::generateMapByExtent(double xCorner1, double
   const Point corner2 = m_mapView->screenToLocation(xCorner2, yCorner2);
   const Envelope extent = Envelope(corner1, corner2);
   const Envelope mapExtent = GeometryEngine::project(extent, SpatialReference::webMercator());
-  const QString tempPath = m_tempDir.path() + "/OfflineMap.mmpk";
-  const QString dataPath = defaultDataPath() + "/ArcGIS/Runtime/Data/tpk";
+  const QString tempPath = m_tempDir.path() + "/OfflineMap.mmpkx";
```

Comment:
I don't think this is correct - AFAIK there is only an MMPK (Mobile Map Package), not MMPKX. Confusingly, there is a MPK and MPKX. Reason being is that ArcMap and Pro both can create map packages, but only Pro can create a Mobile Map Pacakge.

Reply:
Oddly this does work, but it may just be making an MMPK with an invalid file extension though. I'll revert the changes because it's outside the scope of this PR.

Reply:
I think it will let you specify anything, but MMPKX could cause confusion